plugins/run/help-functions

```diff
@@ -29,6 +29,7 @@ fn-help-content() {
   cat <<help_content
     run [-e|--env KEY=VALUE] [--no-tty] <app> <cmd>, Run a command in a new container using the current app image
     run:detached [-e|-env KEY=VALUE] [--no-tty] <app> <cmd>, Run a command in a new detached container using the current app image
+    run:attachable [-e|-env KEY=VALUE] <app> <cmd>, Run a command in a new detached container using the current app image. Container has TTY enabled and is not removed automatically.
```

Comment:
Could this just be a `--force-tty` flag on `run:detached` instead?

Reply:
You are right. Thought I needed to get rid of the DOKKU_CONTAINER_RM too, but it seems to work and is actually better. Will clean this up